src/ast/main.cc

```diff
@@ -0,0 +1,40 @@
+#include "cli.h"
+#include "files.h"
+#include "parser.h"
+#include "sym.h"
+
+// #include "llvm/ExecutionEngine/ExecutionEngine.h"
+// #include "llvm/ExecutionEngine/GenericValue.h"
+// #include "llvm/ExecutionEngine/MCJIT.h"
+// #include <llvm/IR/IRBuilder.h>
```

Comment:
I think those commented out lines can go. I wouldn't use this same main file as the main driver, and would only have it as an AST testing framework. I'm doing the same thing with the MLIR side, `verona-opt`, which will take in MLIR and spit out MLIR or LLVM. Same for the bridge between AST->MLIR.

I think the main compiler should use this AST framework as a library, as well as the conversion bridge between AST and MLIR and the MLIR transformation pipeline, which will also be tested for whole-program support.

If we keep things separate, it's a lot easier to keep the API generic and well tested.